docarray/proto/io/__init__.py

```diff
@@ -22,6 +22,8 @@ def parse_proto(pb_msg: 'DocumentProto') -> 'Document':
             fields[f_name] = [Document.from_protobuf(d) for d in value]
         elif isinstance(value, NdArrayProto):
             fields[f_name] = read_ndarray(value)
+        elif isinstance(value, Struct) and f_name == '_metadata':
+            fields['_metadata'] = MessageToDict(value, preserving_proto_field_name=True)
```

Comment:
This block is redundant with lines of 27-28, can be removed

Reply:
fixed